standard-bridge-custom-token/README.md

```diff
@@ -0,0 +1,62 @@
+
+For an L1/L2 token pair to work on the Standard Bridge the L2 token contract must implement
+[`IL2StandardERC20`](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts/contracts/optimistic-ethereum/libraries/standards/IL2StandardERC20.sol) interface. The standard implementation of that is available in
+[`L2StandardERC20`](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts/contracts/optimistic-ethereum/libraries/standards/L2StandardERC20.sol) contract as part of the `@eth-optimism/contracts` package, see [detailed instructions](../standard-bridge-standard-token/README.md) on using that as your L2 token.
+
+## Customizing the `L2StandardERC20` implementation
+
+Our example here implements a custom token [`L2CustomERC20`](contracts/L2CustomERC20.sol) based on the `L2StandardERC20` but with `8` decimal points, rather than `18`.
+
+For the purpose we import the `L2StandardERC20` from the `@eth-optimism/contracts` package. This standard token implementation is based on the OpenZeppelin ERC20 contract and implements the required `IL2StandardERC20` interface.
+
+```
+import { L2StandardERC20 } from "@eth-optimism/contracts/libraries/standards/L2StandardERC20.sol";
+```
+
+Then the only thing we need to do is call the internal `_setupDecimals(8)` method to alter the token `decimals` property from the default `18` to `8`.
+
+## Deploying the Custom Token
+
+Deployment script is made available under `scripts/deploy-custom-token.js` that you can use to instantiate `L2CustomERC20` both on a local dev node or on `optimism-kovan`.
+
+Once you're ready with a tested kovan deployment, you can request a review via
+[this](https://docs.google.com/forms/d/e/1FAIpQLSdKyXpXY1C4caWD3baQBK1dPjEboOJ9dpj9flc-ursqq8KU0w/viewform) form and we'll consider whitelisting your deployer address on `optimism-mainnet`.
+
+### Prerequisites
+
+You should already have a Hardhat development environment, as explained in
+[the tutorial](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/hardhat).
+
+### The Configuration File
+
+The hardhat config here `hardhat.config.js` is already setup to run against local dev environment, `optimism-kovan` and `optimism-mainnet` networks.
+
+### The .env File
+
+Create an .env file in the root of `standard-bridge-custom-token` folder and add `PRIVATE_KEY` to it. This account is going to be used to call the factory and create your L2 ERC20.
+
+### Update the deploy script
+
+Before you run the `scripts/deploy-custom-token.js` you need to update it with your L1 token details for the different networks in the `TODO` placeholders:
+```
+  l1Token = 'TODO'
+```
+
+### Running the deploy script
+
+Run the following script
+
+```sh
+yarn hardhat run scripts/deploy-custom-token.js --network optimism-kovan
+```
+
+At the end you should get a successful output confirming your token was created and the L2 address:
+
+`L2CustomERC20 deployed to: 0x5CFE8703A62E3a80ab7233263C074698b722d48b`
+
```

Comment:
Maybe a link to the ERC20 send and receive tutorial so they'll know the next step is to verify that ERC20 tokens flow between the two?

Reply:
Good idea